nautobot_ssot/integrations/aci/diffsync/adapters/aci.py

```diff
@@ -73,7 +76,7 @@ def load_tenants(self):
         """Load tenants from ACI."""
         tenant_list = self.conn.get_tenants()
         for _tenant in tenant_list:
-            if not _tenant["name"] in PLUGIN_CFG.get("ignore_tenants"):
```

Comment:
@jdrew82 Would you mind confirming that this is an acceptable change?  Fixed by `inv ruff --fix`